tests/test_playwright.py

```diff
@@ -27,13 +27,10 @@ def test_playwright_click_incorrect():
     plugin = PlaywrightPlugin()
     url = "https://bazos.cz/"
     plugin.call_tool("navigate_to_url", url=url)
-    response = plugin.call_tool(
-        "click_element", selector="a[href='/something-that-doesnt-exist']", index=0
-    )
-    assert (
-        response
-        == "Unable to click on element 'a[href='/something-that-doesnt-exist'] >> visible=1 >> nth=0' as it does not exist"
-    )
+    selector = "a[href='/something-that-doesnt-exist']"
+    response = plugin.call_tool("click_element", selector=selector)
+    expected = f"Unable to click on element '{selector}'"
+    assert response == expected
```

Comment:
I probably broke some of these in my PR, sorry about that. We might consider running the tests as a GH action.

Reply:
Agreed ..

Reply:
All tests work on my PC :D